src/components/modeler/modeler.scss

```diff
@@ -1,3 +1,5 @@
+@import '../../../node_modules/jointjs/dist/joint.min.css';
```

Comment:
I think in scss you can use ~ to refer to node modules:
```scss
@import '~jointjs/dist/joint.min.css';
```

See https://github.com/webpack-contrib/sass-loader#resolving-import-at-rules.